legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/IServiceStoreExecutionExtension.java

```diff
@@ -33,8 +39,15 @@ static List<IServiceStoreExecutionExtension> getExtensions()
         return Lists.mutable.withAll(ServiceLoader.load(IServiceStoreExecutionExtension.class));
     }
 
+    @Deprecated
     default List<Function3<SecurityScheme, HttpClientBuilder, MutableList<CommonProfile>, Boolean>> getExtraSecuritySchemeProcessors()
     {
         return FastList.newList();
     }
+
+    default List<Function5<SecurityScheme, Credential, HttpConnectionBuilder, Identity, CredentialProviderProvider, Boolean>> getExtraCredentialProcessors()
```

Comment:
Is it `credentialProcessors` or `securitySchemeProcessors`?

Reply:
its meant to process/use the obtained credential

Reply:
removed the deprecated function from this file

Reply:
you are right makes sense